syntax = "proto3";

package mypackage;

message DataRequest {
    string name = 1;
    string new_name = 2; 
}

message DataResponse {
    string message = 1;
}

service DataService {
    rpc AddData (DataRequest) returns (DataResponse) {}
    rpc GetData (DataRequest) returns (DataResponse) {}
    rpc UpdateData (DataRequest) returns (DataResponse) {}
    rpc DeleteData (DataRequest) returns (DataResponse) {}
    rpc GetDataList(DataRequest) returns (DataResponse) {} 
}
